.circleci/config.yml

```diff
@@ -217,6 +217,10 @@ jobs:
           command: RUST_BACKTRACE=1 cargo build -j 16 -p language-benchmarks
       - run:
           command: RUST_BACKTRACE=1 cargo build -j 16 -p test-generation
+      - run:
+          command: |
+            rustup target add powerpc-unknown-linux-gnu
```

Comment:
do we want to have an seperate pipeline for powerpc build ?

Reply:
I'll let @sausagee decide that. I'm not sure what impact that has on resource utilization.

Reply:
I think we probably want it at a later time but don't need at the moment. If the dev deps fail to compile in CI, it might be easier for developers to look at them in the same job. 

We may need it eventually when we support a matrix of platforms (amd64, arm) by rustc version (stable, beta, nightly).